# Headlights enabled/disabled outputs

bool high_beam_enabled
bool low_beam_enabled
bool ir_enabled
